### PR TITLE
Intersection: Adds oncoming traffic information via geojson file.

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

Related to 
 - #41 
 - #39 

## Summary
 - Adds oncomming traffic information to the intersection map. Look at the related issues for more information.
 
## Pendings:
- [x] Update output resources out of the template.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)

### CODE REVIEW

File: templates/intersection.xml.em [hunk 1]
```diff
@@ -853,15 +853,46 @@ if crosswalk or stopline:
...
   "features": [
-@[if crosswalk]
+    {
+      "properties": {
+        "Id": "9_0_1",
+        "Type": "Lane",
+        "HasOncomingTraffic": true
+      },
+      "type": "Feature"
+    },
```
Comment (francocipollone) [inline templates/intersection.xml.em:871]: I checked this in `geojson.io` and it complains about that geometry information is missing. @liangfok

Comment (francocipollone) [reply templates/intersection.xml.em:871]: It seems that each feature needs to have a geometry and property information

Comment (francocipollone) [reply templates/intersection.xml.em:871]: We could add some dummy geometry:

```
    {
      "geometry": {
        "coordinates": [
          0, 0
        ],
        "type": "Point"
      },
      "properties": {
        "Id": "9_0_1",
        "Type": "Lane",
        "HasOncomingTraffic": true
      },
      "type": "Feature"
    },
```
Here we can make the point to indicate the start point of the lane

Comment (liangfok) [reply templates/intersection.xml.em:871]: Okay, adding a dummy geometry is fine. Can we include a comment informing future viewers that it is just a dummy, temporary, and currently not used by Maliput?

**Comment by francocipollone** [comment]
Resources were also updated. Ready for review @liangfok

File: templates/intersection.xml.em (hunk 1 above)
Comment (liangfok) [reply templates/intersection.xml.em]: Thanks, I suppose it's not really convenient to only add these has-oncoming-traffic hints to only one map (resources/intersection/intersection_3_3m_width_6m_radius_stopline.xodr)? That's fine as long as the hints actually apply to all of the maps.

Comment (francocipollone) [reply templates/intersection.xml.em]: Correct, I added it to the template, meaning all the intersection maps have this extra information as well.

**Pull request merged**